65.md

```diff
@@ -30,34 +41,50 @@ This NIP doesn't fully replace relay lists that are designed to configure a clie
+- Clients SHOULD post events to at least some (preferably all) of the the p-tag recipient's DM relays.
+- Clients SHOULD post a copy to that client user's own DM relays.
+- If a user has no DM relays, READ relays should be used instead.
 
-When broadcasting an event, Clients SHOULD:
+When seeking events from a user:
 
-- Broadcast the event to the WRITE relays of the author
-- Broadcast the event all READ relays of each tagged user
+- Clients SHOULD seek them from at least some of that user's WRITE relays.
+
+When seeking events about a user (where the user was tagged):
+
+- Clients SHOULD seek them from all of that user's READ relays.
+
+When seeking **DM**s for a user:
+
+- Clients SHOULD seek them from all of that user's DM and READ relays.
+
+In general, when interacting with a user's own relays, clients should interact with all of them (write to all WRITE relays, read from all READ and DM relays).  When interacting with other user's relays, clients need only interact with a subset of them (read from some of their WRITE relays, write to some of their READ and DM relays).
 
 ## Motivation
 
-The old model of using a fixed relay list per user centralizes in large relay operators: 
+The old model of using a fixed relay list per user centralizes in large relay operators:
 
   - Most users submit their posts to the same highly popular relays, aiming to achieve greater visibility among a broader audience
   - Many users are pulling events from a large number of relays in order to get more data at the expense of duplication
   - Events are being copied between relays, oftentimes to many different relays
-  
-This NIP allows Clients to connect directly with the most up-to-date relay set from each individual user, eliminating the need of broadcasting events to popular relays. 
+
+This NIP allows Clients to connect directly with the most up-to-date relay set from each individual user, eliminating the need of broadcasting events to popular relays.
 
 ## Final Considerations
 
-1. Clients SHOULD guide users to keep `kind:10002` lists small (2-4 relays). 
+1. Clients SHOULD guide users to keep `kind:10002` lists small (2-6 relays).
 
-2. Clients SHOULD spread an author's `kind:10002` event to as many relays as viable. 
+2. Clients SHOULD advertise an author's `kind:10002` event to as many relays as viable.
 
 3. `kind:10002` events should primarily be used to advertise the user's preferred relays to others. A user's own client may use other heuristics for selecting relays for fetching data.
 
-4. DMs SHOULD only be broadcasted to the author's WRITE relays and to the receiver's READ relays to keep maximum privacy. 
+4. DMs SHOULD only be broadcasted to the recipient's and author's DM relays. Previously this NIP specified for them to be broadcast to the author's WRITE relays but that is less secure. Clients may have to look for them in both places, but should post them only to DM relays.
```

Comment:
There is no need to add historical reasoning. I would just keep the guidance for what should happen from now on.

Actually... since this guidance seems to be duplicated from the text above, there might not be a need for this 4th point here anymore. This point was here exactly because there was no explicit pattern for DMs before.

Reply:
removed.